apps/meteor/client/views/admin/manageSubscription/components/cards/CountSeatsCard.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Icon, Skeleton } from '@rocket.chat/fuselage';
+import type { ReactElement } from 'react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { useSeatsCap } from '../../../../../../ee/client/views/admin/users/useSeatsCap';
+import FeatureUsageCard from '../FeatureUsageCard';
+
+const CountSeatsCard = (): ReactElement => {
+	const { t } = useTranslation();
+	const seatsCap = useSeatsCap();
+
+	const card = {
+		title: t('Seats'),
+		infoText: t('CountSeats_InfoText'),
+	};
+
+	const maxSeats = seatsCap?.activeUsers || 0;
+
+	return (
+		<FeatureUsageCard title={card.title} infoText={card.infoText}>
+			{seatsCap ? (
+				<Box fontScale='h1'>
+					<Icon color='font-annotation' name='user' size={40} mie={4} />
```

Comment:
`<Icon />` shouldn't be children of `<h1>` element

Reply:
why not?

Reply:
I think I was trippy when I said that, but sometimes you could have issues align, so I commonly avoid this